lib/conversation.js

```diff
@@ -1,11 +1,62 @@
 'use strict';
```

Comment:
Notó que faltan los Type Definition

https://fizzmod.atlassian.net/wiki/spaces/JV2/pages/2239299713/Type+Definition+en+package